packages/utils/src/format.ts

```diff
@@ -45,14 +45,10 @@ export function formatBigDecimal(numerator: bigint, denominator: bigint, maxDeci
 const MAX_DECIMAL_FACTOR = BigInt("100000");
 
 /**
- * Format wei as ETH, with up to 5 decimals
- *
- * if suffix is true, append ' ETH'
+ * Format wei as ETH, with up to 5 decimals and append ' ETH'
  */
-export function prettyWeiToEth(wei: bigint, suffix = false): string {
```

Comment:
Removed option to omit suffix as it is unused and would just lead to inconsistent logs. 

We can discuss how we wanna log ETH values (with / without suffix), I am fine with either as long as we do it consistently.